scripts/e2e-test-local-cluster-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "us-east-1": [
+    {
+      "dbConfig": {
+        "identifier": ""
+      }
+    }
+  ],
+  "us-west-2": [
+    {
+      "dbConfig": {
+        "identifier": "YMmwoSDbAQ"
```

Comment:
Clear this value to reduce confusion. Alternately, set it to a clearly bogus value to indicate that it must be replaced. Whatever you decide, please make sure to do the same for the us-east-1 value above

Reply:
Replace with sample values like Tim suggested and can we rename this file to `e2e-test-local-cluster-config.sample.json` and also add a `.gitignore` entry for `e2e-test-local-cluster-config.json` so developers don't commit it accidentally?